dev-packages/e2e-tests/test-applications/solidjs/src/index.tsx

```diff
@@ -7,7 +7,9 @@ import PageRoot from './pageroot';
 import { routes } from './routes';
 
 Sentry.init({
-  dsn: import.meta.env.PUBLIC_E2E_TEST_DSN,
+  dsn:
+    import.meta.env.PUBLIC_E2E_TEST_DSN ||
+    'https://3b6c388182fb435097f41d181be2b2ba@o4504321058471936.ingest.sentry.io/4504321066008576',
```

Comment:
Do we need this fallback or is it a leftover from local testing?

Reply:
Leftover from local testing. I noticed the react-19 test app also has that committed. Can remove if we mind, it's just annoying locally.

Reply:
Removed